lesson19/src/js/tab.js

```diff
@@ -0,0 +1,173 @@
+        fragmentTopicImg.appendChild(topicsImg);
+    }
+
+    contentsBox.appendChild(imageArea).appendChild(fragmentTopicImg);
+}
+
+const renderArticlesSection = (newsData) => {
+    const contentsBox = document.getElementById("js-contents-box");
+    const articleArea = createAttributedElements({
+        tag:"div",
+        valuesByAttributes:{class:"contents-item articles"}
+    });
+
+    const fragmentAritcle = document.createDocumentFragment(); 
+    for (const topics of newsData){
+        const articleList = createAttributedElements({
+            tag:"ul",
+            valuesByAttributes:{class:"js-article-list article-list"}
+        });
+        fragmentAritcle.appendChild(articleList)
+        .appendChild(createArticleTitleFlagment(topics));
+    }
+
+    contentsBox.appendChild(articleArea).appendChild(fragmentAritcle); 
+}
+
+const createArticleTitleFlagment = ({articles}) => {
+    const articleTitleFlagment = document.createDocumentFragment(); 
+
+    for(const article of articles ){
+        const articleItem = createAttributedElements({
+            tag:"li",
+            valuesByAttributes:{class:"js-article-list__item article-list__item"}
+        });
+
+        const articleLink = createAttributedElements({
+            tag:"a",
+            valuesByAttributes:{class:"js-article-link"},
+        });
+        
+        const articleTitle = createAttributedElements({
+            tag:"p",
+            valuesByAttributes:{class:"article-title"},
+            str:article.title
+        });
+
+        articleTitleFlagment.appendChild(articleItem).appendChild(articleLink).appendChild(articleTitle);
+
+        isNewAriticle(article.date) && showNewIcon(articleItem);
+        (article.comment.length) && showCommentIconWithNumber(articleItem, article.comment.length);
+    }
+
+    return articleTitleFlagment;
+}
+
+const isNewAriticle = (articleDate) => {
+    const SpecificPeriod = 3;
+    const dateDifferencial = differenceInDays(new Date(), new Date(articleDate));
+
+    return (SpecificPeriod > dateDifferencial);
+}
+
+const showNewIcon = (articleLink) => {
+    const newIcon = createAttributedElements({
+        tag:"span",
+        valuesByAttributes:{class:"new-icon"},
+        str:"New"
+    });
+
+    articleLink.appendChild(newIcon);
+}
+
+const showCommentIconWithNumber = (articleLink,commentLength) => {
+    const commentIcon = createAttributedElements({
+        tag:"img",
+        valuesByAttributes:{class:"comment-icon", src:"img/comment.svg"}
+    });
+
+    const commentNumber = createAttributedElements({
+        tag:"span",
+        valuesByAttributes:{class:"comment-number"},
+        str:commentLength
+    });
+
+    articleLink.appendChild(commentIcon);
+    articleLink.appendChild(commentNumber);
+}
+
+const tabItems = [ "js-topics-list__item", "js-topics-img__item", "js-article-list" ];
+
+const activateFirstTab = (newsData) => {
+    const index = newsData.findIndex((value) => value.isActive === true);
```

Comment:
IMO
```suggestion
    const index = newsData.findIndex((value) => value.isActive);
```

Reply:
exactly.
Since isActive is a true process, there is no need to make the value explicit.
I fixed.
https://github.com/nakagawamai/js-lesson/pull/21/commits/fa96ee27a98d89c82c972c2c8c28a8cef236ff87